lib/spack/docs/contribution_guide.rst

```diff
@@ -189,6 +189,7 @@ Building the documentation requires several dependencies, all of which can be
 installed with Spack:
 
 * sphinx
+* sphinxcontrib-programoutput
```

Comment:
A `py-sphinxcontrib-programoutput` package coming soon to a Spack near you: #4267.